scripts/quests/windurst/RNG_AF1_Sin_Hunting.lua

```diff
@@ -0,0 +1,161 @@
+                    elseif quest:getVar(player, 'Prog') == 5 then
+                        return quest:progressEvent(527)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [527] = function(player, csid, option, npc)
+                    if quest:complete(player) then
+                        player:delKeyItem(xi.ki.PERCHONDS_ENVELOPE)
+                    end
+                end,
+            },
+        },
+
+        [xi.zone.RANGUEMONT_PASS] =
+        {
+            ['Perchond'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(3, 0, xi.items.GLITTERSAND)
+                    else
+                        return quest:event(2)
+                    end
+                end,
+
+                onTrade = function(player, npc, trade)
+                    if
+                        npcUtil.tradeHasExactly(trade, xi.items.GLITTERSAND) and
+                        quest:getVar(player, 'Prog') == 2
+                    then
+                        return quest:progressEvent(5)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [3] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 2)
+                end,
+
+                [5] = function(player, csid, option, npc)
+                    player:tradeComplete()
+                    quest:setVar(player, 'Prog', 3)
+                    npcUtil.giveKeyItem(player, xi.ki.PERCHONDS_ENVELOPE)
+                end,
+            },
+        },
+
+        [xi.zone.JUGNER_FOREST] =
+        {
+            ['Alexius'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 3 then
+                        return quest:progressEvent(10)
+                    elseif quest:getVar(player, 'Prog') == 4 then
+                        return quest:event(11)
+                    elseif quest:getVar(player, 'Prog') == 5 then
+                        return quest:event(12)
+                    end
+                end,
+            },
+
+            ['qm2'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 4 and
+                        IsMoonFull()
+                    then
+                        return quest:progressEvent(13, 0, xi.items.GLITTERSAND)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 4)
+                end,
+
+                [13] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 5)
```

Comment:
are players suppposed to get glittersand here?

Reply:
glittersand is a drop from the evil weapons.